python/ql/src/semmle/python/frameworks/internal/AutomatedModeling.qll

```diff
@@ -0,0 +1,1901 @@
+    spec = "Django::Views::View" and
+    qualifier = "openstack_dashboard.dashboards.project.backups.views.BackupDetailView"
+    or
+    spec = "Django::Views::View" and
+    qualifier = "openstack_dashboard.dashboards.project.backups.views.RestoreBackupView"
+    or
+    spec = "Django::Views::View" and
+    qualifier = "openstack_dashboard.dashboards.identity.projects.views.IndexView"
+    or
+    spec = "Django::Views::View" and
+    qualifier = "openstack_dashboard.dashboards.identity.projects.views.ProjectUsageView"
+    or
+    spec = "Django::Views::View" and
+    qualifier = "openstack_dashboard.dashboards.identity.projects.views.CreateProjectView"
+    or
+    spec = "Django::Views::View" and
+    qualifier = "openstack_dashboard.dashboards.identity.projects.views.UpdateProjectView"
+    or
+    spec = "Django::Views::View" and
+    qualifier = "openstack_dashboard.dashboards.identity.projects.views.UpdateQuotasView"
+    or
+    spec = "Django::Views::View" and
+    qualifier = "openstack_dashboard.dashboards.identity.projects.views.DetailProjectView"
+    or
+    spec = "Django::Views::View" and
+    qualifier = "openstack_dashboard.contrib.developer.theme_preview.views.IndexView"
+    or
+    spec = "Django::Views::View" and
+    qualifier = "openstack_dashboard.dashboards.project.volumes.views.VolumesView"
+    or
+    spec = "Django::Views::View" and
+    qualifier = "openstack_dashboard.dashboards.project.volumes.views.DetailView"
+    or
+    spec = "Django::Views::View" and
+    qualifier = "openstack_dashboard.dashboards.project.volumes.views.CreateView"
+    or
+    spec = "Django::Views::View" and
+    qualifier = "openstack_dashboard.dashboards.project.volumes.views.ExtendView"
+    or
+    spec = "Django::Views::View" and
+    qualifier = "openstack_dashboard.dashboards.project.volumes.views.CreateSnapshotView"
+    or
+    spec = "Django::Views::View" and
+    qualifier = "openstack_dashboard.dashboards.project.volumes.views.UploadToImageView"
+    or
+    spec = "Django::Views::View" and
+    qualifier = "openstack_dashboard.dashboards.project.volumes.views.CreateTransferView"
+    or
+    spec = "Django::Views::View" and
+    qualifier = "openstack_dashboard.dashboards.project.volumes.views.AcceptTransferView"
+    or
+    spec = "Django::Views::View" and
+    qualifier = "openstack_dashboard.dashboards.project.volumes.views.ShowTransferView"
+    or
+    spec = "Django::Views::View" and
+    qualifier = "openstack_dashboard.dashboards.project.volumes.views.UpdateView"
+    or
+    spec = "Django::Views::View" and
+    qualifier = "openstack_dashboard.dashboards.project.volumes.views.EditAttachmentsView"
+    or
+    spec = "Django::Views::View" and
+    qualifier = "openstack_dashboard.dashboards.project.volumes.views.RetypeView"
+    or
+    spec = "Django::Views::View" and
+    qualifier = "openstack_dashboard.dashboards.project.volumes.views.EncryptionDetailView"
+    or
+    spec = "Django::Views::View" and
```

Comment:
It would be shorter and clearer to write
```
spec = "Django::Views::View" and qualifier in
[
"openstack_dashboard.dashboards.project.volumes.views.DownloadTransferCreds",
"horizon.tables.views.MultiTableView",
"horizon.tables.views.MixedDataTableView",
...
]
```

Reply:
Agreed. Since this data is generated I just wanted to keep it simple. As mentioned in the comments, I think this data should live in some external file in the future.